### ISSUE TITLE
[BUG] Amount of tab args doesn't work but it's a list

### ISSUE THREAD

**Issue by ImNotStable** [ShaneBeee/SkBee#328]
**Describe the bug**   
I can't use the expression `number of tab args`, but I can use
`
set {_test::*} to tab args
broadcast "%{_test::1}%"
`

**To Reproduce**   
Please include clear steps to reproduce the behavior
in a tab complete event use this code
`broadcast "%number of tab args%"`
and you'll get this error
'tab complete args' can only ever have one value at most, thus the 'amount of ...' expression is useless. Use '... exists' instead to find out whether the expression has a value.
Line: broadcast "%number of tab args%"

**Expected behavior**   
For the expression to work

**Screenshots/Errors**   
If applicable, add screenshots/errors to help explain your problem.

**Version Info:**
```
--- [SkBee Loading Info] ---
- [NBTAPI] Found Spigot: v1_18_R2! Trying to find NMS support
- [NBTAPI] NMS support 'MC1_18_R2' loaded!
- NBT Elements successfully loaded
- Recipe Elements disabled via config
- Scoreboard Elements successfully loaded
- Scoreboard Objective Elements successfully loaded
- Team Elements successfully loaded
- Bound Elements disabled via config
- Text Component Elements successfully loaded
- Pathfinding Elements successfully loaded
- Structure Elements disabled via config
- It looks like another addon registered 'itemflag' already.
- You may have to use their Item Flags in SkBee's 'Hidden Item Flags' expression.
- Other Elements successfully loaded
- Virtual Furnace Elements disabled via config
- World Creator Elements disabled via config
- Game Event Elements successfully loaded
- BossBar Elements successfully loaded
- Statistic Elements successfully loaded
- It looks like another addon registered 'profession' already.
- You may have to use their profession in SkBee's 'Villager Profession' expression.
- It looks like another addon registered 'villagertype' already.
- You may have to use their villagertype in SkBee's 'Villager Type' expression.
- Villager Elements successfully loaded
- Advancement Elements successfully loaded
- World Border Elements successfully loaded
- Particle Elements successfully loaded
- Minecraft Tag elements successfully loaded
- RayTrace elements successfully loaded
- Plugin is not up to date!
-  - Current version: v2.7.1
-  - Available update: v2.7.3
-  - Download available at: https://github.com/ShaneBeee/SkBee/releases
--- [Server Info] ---
Server Version: git-Paper-388 (MC: 1.18.2)
Skript Version: 2.6.4
Skript Addons:
- Beamer v1.0.3
- skript-gui v1.3
- DiSky v4.8.0
- Diskuise v0.3.3
- skript-reflect v2.3
- skUtilities v0.9.2
- skRayFall v1.9.28
- SkQuery v4.1.7
- SkJade v1.4.2
- Khoryl v1.0.6
- Skcrew v1.4
SkBee Version: 2.7.1
SkBee Website: https://github.com/ShaneBeee/SkBee
```

**Comment by ShaneBeee** [comment]
Can you please try use a current version of SkBee and see if the issue persists.
To the best of my knowledge this was already fixed in 2.7.2, and I see you are using 2.7.1
<img width="1448" alt="Screenshot 2023-01-27 at 3 10 21 PM" src="https://user-images.githubusercontent.com/20008482/215223001-9ec22a82-bc81-4b9d-842b-40d267c6ba9d.png">

**Comment by ImNotStable** [comment]
Works, my bad for not checking updates.
